docs/manual/introduction/Optimizations.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<base href="../../" />
+		<script src="list.js"></script>
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		<h1>[name]</h1>
+
+		<p class="desc">
+			This article shows how to optimize your three.js scene.
+		</p>
+
+		<h2>Metrics</h2>
+
+		<p>
+			Before starting, we need to define a some metrics, and how to monitor them. There are two important factors
+			that will affect your performances:
+
+			<ul>
+				<li>CPU: affects the number of frames per seconds (FPS) which represents how fluid is your scene</li>
+				<li>Memory: affects the memory usage which can be restricted on low devices like smartphones</li>
+			</ul>
```

Comment:
This is too simplified I think - there is CPU, GPU, then CPU memory and GPU memory. 

However I would just leave this section out for now.